src/components/item/item.ts

```diff
@@ -354,4 +362,24 @@ export class Item {
       icon.addClass('item-icon');
     });
   }
+
+  /**
+   * @private
+   */  
+  height(): number {
+    return this._elementRef.nativeElement.offsetHeight;
```

Comment:
Where does this get called from? I worry that `height()` gets called a lot from every change detection, which would numerous and unpredictable DOM reads.